### PR TITLE
remove `.global` from docs

### PR DESCRIPTION
**Pull request by Pothulapati**
Fixes linkerd/linkerd2#5733

As the global field is removed on the latest edges, This PR
updates the docs to remove its use.

The plan is to merge this before the release happens.

Signed-off-by: Tarun Pothulapati <tarunpothulapati@outlook.com>

### CODE REVIEW

**Review by alpeb — COMMENTED** [review]
I found a few more occurrences of `global`:

- linkerd.io/content/2/features/cni.md
- linkerd.io/content/2/tasks/external-prometheus.md
- linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md

**Comment by alpeb** [comment]
Fixes linkerd/linkerd2#5733

**Comment by cpretzer** [comment]
@Pothulapati just to be clear, this should be merged before the `2.10` release?

**Comment by adleong** [comment]
@Pothulapati can you switch the base branch of this PR to be `release/stable-2.10`?

**Comment by Pothulapati** [comment]
> linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md

@alpeb I think that occurance is for < 2.9 which I think is valid with the older json config that we had. 🤔

**Comment by alpeb** [comment]
> > linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md
> 
> @alpeb I think that occurance is for < 2.9 which I think is valid with the older json config that we had.

I don't understand. Is that doc specific to 2.9 only? Anyways, I addressed that document separately in #925

File: linkerd.io/content/2/tasks/external-prometheus.md [hunk 1]
```diff
@@ -3,8 +3,9 @@ title = "Bringing your own Prometheus"
 description = "Use an existing Prometheus instance with Linkerd."
 +++
 
-Even though Linkerd comes with its own Prometheus instance, there can be cases
-where using an external instance makes more sense for various reasons.
+Even though linkerd-viz extension comes with its own Prometheus instance,
+there can be casesz where using an external instance makes more
```
Comment (alpeb) [inline linkerd.io/content/2/tasks/external-prometheus.md:7]:
```suggestion
there can be cases where using an external instance makes more
```

File: linkerd.io/content/2/tasks/external-prometheus.md [hunk 2]
```diff
@@ -132,32 +133,31 @@ with direct values for the below configuration to work.
...
 ```yaml
-global:
-  prometheusUrl: existing-prometheus.xyz:9090
+prometheusUrl: existing-prometheus.xyz:9090
 ```
 
-Once applied, this configuration is persistent across upgrades, without having
-the user passing it again. The same can be overwritten as needed.
+Once applied, this configuration is not persistent across installs.
+The same has to be passed again by the user during re-installs, upgrades, etc.
```
Comment (alpeb) [inline linkerd.io/content/2/tasks/external-prometheus.md:158]: Great point, I had forgotten viz config doesn't persist!

File: linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md [hunk 3]
```diff
@@ -127,7 +127,7 @@ kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.global}' \
 
 # For Linkerd >= 2.9.0:
 kubectl -n linkerd get cm linkerd-config -o=jsonpath='{.data.values}' \
-  | yq r - global.identityTrustAnchorsPEM > original-trust.crt
+  | yq r - identityTrustAnchorsPEM > original-trust.crt
```
Comment (alpeb) [inline linkerd.io/content/2/tasks/manually-rotating-control-plane-tls-credentials.md:130]:
```suggestion
  | yq -r .identityTrustAnchorsPEM > original-trust.crt
```

**Review by alpeb — APPROVED** [review]
:+1:

**Review by adleong — APPROVED** [review]

**Pull request merged**